component/outbound/dialer/dialer.go

```diff
@@ -39,6 +39,12 @@ type GlobalOption struct {
 	CheckTolerance    time.Duration
 	CheckDnsTcp       bool
 	AllowInsecure     bool
+	TlsImplementation string
+	Utls              Utls
```

Comment:
Can we remove this to reduce parameter number and give it a default value?

Reply:
`TlsImplementation·`should be held so that we can easily add other TLS implementations. But I can change `Utls.Imitate` to `UtlsImitate`. I think it should be completely up to the user to choose the `Client Hello ID` so that the user knows what they are doing.

Reply:
Great. I think UtlsImitate is better! Thanks ~

Reply:
By the way, could you please add some annotations(and all option values) in example.dae and also in config/desc.go?

Reply:
Of course.